### ISSUE TITLE
Web Feature Missing in latest cuckoo?

### ISSUE THREAD

**Issue by sxu55** [cuckoosandbox/cuckoo#1612]
Hi, I just started using the latest Cuckoo and noticed that the latest version (2.0) has a revamped beautiful web interface. But I really miss the old report html page where you can see the list of accessed/modified filenames , registries, and services (like the Summary in this link: https://malwr.com/analysis/ZDVlZmM3MmU5ZjIzNDg5Yzg5NTA3ZjlkODMwN2YyOTc/). It seems the new web interface simply got rid of these reports. Will it be possible to bring this old feature back?

**Comment by SparkyNZL** [comment]
Isn't this now under Behavioral analysis ?

**Comment by sxu55** [comment]
I didn't see one in my behavioral analysis page. I know that the behavioral analysis page can also show these by clicking filters, I am just wondering if the main reporting page can have a concise list of these stuffs like ones in the Malwr's page... maybe I am just too lazy ...

**Comment by jbremer** [comment]
That's correct, yes, @sxu55. The reasoning behind this - from my side - is that there's simply too much information available for a "quick summary overview". I'd be up for showing key information on the summary page, but showing potentially thousands of entries simply doesn't make sense.
If you have thoughts on this - e.g., better filtering of dummy information and/or additional Signatures to identify what should be shown - do let us know!

**Comment by sxu55** [comment]
Thanks! I agree with you that when the list is long, there should be a good filtering mechanism. The reason I propose this is because I spotted some recent ransomware which encrypt documents in a random fashion (both in scan and in encryption, some do not even exceed the cuckoo file modification limit so no flag is raised). In this case, a concise list of modified file might help users to quickly identify potential threats.

There is another issue though... Recent ransomwares like Jaff exhibit many more behaviors compared to old ones like Cerber. This directly result in frequent MongoDB 16MB limit error in the web report. In fact, to let some samples run properly in current Cuckoo 2.0.3, I have to limit their run time down to 30secs to prevent the explosion of logs and failure in MongoDB. I know this issue has long been discussed here, but just wondering if there is any temporary work around for this issue, or is this issue currently being addressed?

**Comment by Tigzy** [comment]
Now that you say it, it's true that this information has been removed. 
Maybe just showing the list of created items ? (They're usually fewer than modified stuff)
And it's to quickly see the malware location and startup entries.

**Comment by RANGERBEE** [comment]
concur..
